### PR TITLE
feat: add Mantle chain

### PR DESCRIPTION
**Pull request by myz1237**
https://lifi.atlassian.net/browse/LF-8063

### CODE REVIEW

File: src/chains/base.ts [hunk 1]
```diff
@@ -111,6 +111,7 @@ export enum ChainId {
   BAS = 8453,
   SCL = 534352,
   MOD = 34443,
+  MNT = 5000,
```
Comment (melianessa) [inline src/chains/base.ts:114]: I think we also need to add chainKey for the new chain

**Review by melianessa — CHANGES_REQUESTED** [review]

File: src/chains/base.ts (hunk 1 above)
Comment (myz1237) [reply src/chains/base.ts:114]: Thanks, forgot this part, damn

**Comment by myz1237** [comment]
This pr will be a draft, since our contract is not deployed yet, will merge it next Monday

**Comment by myz1237** [comment]
Contract deployed, gonna ship it

**Pull request merged**